src/components/views/dialogs/TchapLogoutDialog.tsx

```diff
@@ -0,0 +1,294 @@
+                                            {},
+                                            {
+                                                b: (sub) => <b>{ sub }</b>,
+                                            },
+                                        ) }
+                                    </p>
+                                    <button
+                                        className="danger"
+                                        onClick={this.onLogoutConfirm}
+                                    >
+                                        { _t("Sign me out") }
+                                    </button>
+                                </div>
+                                <div className="tc_TwoColumn_block_content">
+                                    <div className="tc_TwoColumn_block_image">
+                                        <img
+                                            src={ExportLogoImage}
+                                            alt="Export logo"
+                                            width="70"
+                                        />
+                                    </div>
+                                    <p>
+                                        { _t(
+                                            "<b>You don't have another device connected to Tchap?</b>"
+                                            + " Back up your Tchap Keys. These keys will unlock current messages,"
+                                            + " but not those received after saving.",
+                                            {},
+                                            {
+                                                b: (sub) => <b>{ sub }</b>,
+                                            },
+                                        ) }
+                                    </p>
+                                    <button
+                                        className="mx_Dialog_primary"
+                                        onClick={this.onExportE2eKeysClicked}
+                                    >
+                                        { _t("Save my keys") }
+                                    </button>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                );
+            }
+            // Not quite a standard question dialog as the primary button cancels
+            // the action and does something else instead, whilst non-default button
+            // confirms the action.
+            return (
+                <BaseDialog
+                    title={_t("Save your Tchap Keys before logging out")}
+                    contentId="mx_Dialog_content"
+                    hasCancel={true}
+                    onFinished={this.onFinished}
+                >
+                    { dialogContent }
+                </BaseDialog>
+            );
+        } else {
```

Comment:
dans notre cas, on veut afficher le dialog complet dans tous les cas. On a pas besoin de ce "are you sure you want to sign out". Donc on peut enlever le `if (this.state.shouldLoadBackupStatus)` ligne 144. (et du coup enlever les fonctions pas utilisees au besoin)